modules/cd-18/module.yaml

```diff
@@ -0,0 +1,59 @@
+schema_version: 1
+
+name: "eap-cd-18"
+version: "1.0"
+description: "JBoss Enterprise Application Platform CD 18 modular zip install"
+labels:
```

Comment:
@luck3y , there is a module https://github.com/jboss-container-images/jboss-eap-7-image/blob/eap-cd-dev/modules/eap-cd-env/18.0/module.yaml that contains the env and labels used by the galleon image. You could depend on this module and not define env/labels?

Reply:
+1, done.